snake/snake.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/python3.11
+
+class SnakeGame:
+
+    def __init__(self, width: int, height: int, food: list[list[int]]):
+        self.snake_loc = [(0,0)]
+        self.width = width
+        self.height = height
+        self.food = food
+
+
+    def move(self, direction: str) -> int:
+        score = len(self.snake_loc) - 1
+        if direction == 'R':
```

Comment:
I would advice against "if ladder".
My suggestions:
1.  Read about  manhattan distance https://xlinux.nist.gov/dads/HTML/manhattanDistance.html
2. We can formulate - 
new_location = current_location + (delta_x, delta_y)
3. Create map, where key is direction, and value is delta touple (delta_x, delta_y)
4. Use the map instead of if-ladder

Reply:
Fixed, thanks